client/gutenberg/extensions/tiled-gallery/gallery-image.jsx

```diff
@@ -0,0 +1,168 @@
+/** @format */
+
+/**
+ * This component is originally from Gutenberg Gallery block:
+ * @link https://github.com/WordPress/gutenberg/blob/1cd604df7e9017e0dbe3ab64897ac3af35ca35c5/packages/block-library/src/gallery/gallery-image.js
+ */
```

Comment:
We should try `import galleryImage from @wordpress/block-library/😬` at least to see what happens.